aiida_restapi/aiida_db_mappings.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yeh I guess there were some open questions for me, regarding what will be required for these model when used with the restapi. Currently, there is the `get_entities` classmethod, but this is probably not the only thing that is required for these models to be properly used in the REST API (if indeed we do actually want the rest api endpoints 😬), and so I kind of didn't want the graphql to "inherit" such complexities.
Potentially what I would want then, is for models.py to still be separate, but inherit from both the aiida_db_mappings models and an AiiDAmodel mixin. But maybe this is other-thinking it 🤷

Reply:
Happy with the mixin solution if you prefer this

Reply:
eurgh, can we leave this for another PR, otherwise I can't finish this PR because there is question marks, like should these models be specified for queries or creations, or should they be separate, e.g. I have specified `id: int = Field(description="Unique id (pk)")`, because you should never have an id returned as None from a stored entity, but in models.py it is set as `id: Optional[int] = Field(description="Id of the object")`, essentially to stop fastapi from failing when you don't supply an id. But really, for the creation model, there should be no `id` field, because you are "not allowed" to manually specify the id (same for uuid, ctime, ctime etc)

Reply:
basically I think it needs more thought